src/interface.rs

```diff
@@ -0,0 +1,23 @@
+wit_bindgen_wasmtime::export!("src/host.wit");
+wit_bindgen_wasmtime::import!("src/guest.wit");
+
+#[derive(Default)]
+pub struct Host;
+
+impl host::Host for Host {
+    fn get_one_with_args(
+        &mut self,
+        query: &str,
+        args: Vec<host::ValueParam<'_>>
```

Comment:
Where does `ValueParam` and `ValueResult` get defined from? Is it accessible as part of the